scipy/sparse/_sputils.py

```diff
@@ -390,14 +390,16 @@ def is_pydata_spmatrix(m) -> bool:
 
 
 def convert_pydata_sparse_to_scipy(
-    arg: Any, target_format: Optional[Literal["csc", "csr"]] = None
+    arg: Any,
+    target_format: Optional[Literal["csc", "csr"]] = None,
+    accept_fv: Any = None,
 ) -> Union[Any, "sp.spmatrix"]:
     """
     Convert a pydata/sparse array to scipy sparse matrix,
     pass through anything else.
     """
     if is_pydata_spmatrix(arg):
-        arg = arg.to_scipy_sparse()
+        arg = arg.to_scipy_sparse(accept_fv=accept_fv)
```

Comment:
This looks like it's going to break for `sparse<0.15.4`.

Reply:
Done!